tdxhydrorapid/inputs.py

```
@@ -24,11 +30,12 @@
     'dissolve_branches',
     'prune_branches',
     'rapid_input_csvs',
+    'river'
     'concat_tdxregions',
```

Ruff (0.8.2)

Undefined name `riverconcat_tdxregions` in `__all__`

(F822)